sycl/doc/extensions/KernelProperties/KernelProperties.asciidoc

```diff
@@ -125,55 +119,55 @@ namespace ext {
 namespace oneapi {
```

Comment:
Didn't we decide that the first implementation of `properties` will be experimental?  If so, I presume the kernel properties should also be experimental, so this namespace should change to `sycl::ext::oneapi::experimental`.

See my similar comment: https://github.com/intel/llvm/pull/5338/files#r791135338

Reply:
Yes, I think you're right.  I'll update this.

Reply:
Fixed in ecb211a9176ef8b6f28b299bcbeccea12d0364d9.